openid4vc-high-assurance-interoperability-profile-sd-jwt-vc-1_0.md

```diff
@@ -272,11 +267,79 @@ Note: The issuer MAY decide to support both options. In which case, it is at the
 
 A Credential Format Profile for Credentials complying with IETF SD-JWT VCs [@!I-D.ietf-oauth-sd-jwt-vc] is defined in Annex A.3 of [@!OIDF.OID4VCI] and Annex A.4 of [@!OIDF.OID4VP].
 
+## SD-JWT VCDM
```

Comment:
We should clearly say something about the goals of SD-JWT VCDM and who can consume it. For example, we should define what of the following scenarios are possible:
- SD-JWT VC processor without JSON-LD processing
- SD-JWT VC processor with JSON-LD processing
- W3C VCDM 2.0 processor (which requires JSON-LD), with securing mechanism verification. This might be an issue because of the external context definitions will be required, and because SD-JWT VC has to be defined as a W3C VCDM 2.0 securing mechanism.
- W3C VCDM 2.0 processor without securing mechanism verification; just do SD-JWT VC verification. This will also require external context definitions for all public/private JWT claims which are not defined in the W3C VCDM 2.0 context already. 
- any other options? 

After this PR got merged: https://github.com/w3c/vc-data-model/pull/1520, VCDM 2.0 requires all claims to be defined explicitly in dedicated `@context` definitions referenced by the `@context` array. This is required because all VCDM 2.0 credentials have to be JSON-LD compliant. Before this PR, this was not strictly required (although kind of recommended). Not having these properly defined in these context definitions would break VCDM 2.0 processors. The VCDM 2.0 context does have definitions for all the standard SD-JWT terms but does not have definitions of SD-JWT VC claims such as `status`, `schema` etc. Previously all these claims got added the default context as `issuer-dependent` terms via the `"@vocab": "https://www.w3.org/ns/credentials/issuer-dependent#"` statement in the W3C VCDM 2.0 context.

Reply:
It is possible to add a default vocab in HAIP but it should be noted that this MUST NOT be done if Data Integrity Proofs are used. I'm not sure yet if this is a good idea though.

However, we would need to use a different IRI for the value of `@vocab` since https://www.w3.org/ns/credentials/issuer-dependent#" does not go anywhere.